src/strapPy/data_vis.py

```diff
@@ -81,7 +81,7 @@ def histogram_ci_plot(sample, rep, bin_size = 30, n="auto", ci_level=0.95,
     
 
 
-def summary_tables(stat, precision=2, estimator=True, alpha=True):
+def create_tables(stat, precision=2, estimator=True, alpha=True):
```

Comment:
```suggestion
def tabulate_stats(stat, precision=2, estimator=True, alpha=True):
```

@voremargot I personally feel tabulate_stats summarizes the function well but I am open to your feedback.